crates/daphne/src/roles/mod.rs

```diff
@@ -1516,7 +1513,18 @@ mod test {
         });
     }
 
-    async fn e2e_taskprov_prio2(version: DapVersion) {
+    async fn e2e_taskprov_prio2_draft09(version: DapVersion) {
+        e2e_taskprov(
+            version,
+            VdafConfig::Prio2Draft09 { dimension: 10 },
+            DapMeasurement::U32Vec(vec![1; 10]),
+        )
+        .await;
+    }
+
+    async_test_versions! { e2e_taskprov_prio2_draft09 }
+
+    async fn e2e_taskprov_prio2_latest(version: DapVersion) {
         e2e_taskprov(
             version,
             VdafConfig::Prio2 { dimension: 10 },
```

Comment:
remerge these two tests and use the passed in version to determine the prio version to use

Reply:
I don't want to tie Prio too tightly to DAP, especially in the tests.